.rubocop.yml

```diff
@@ -1,22 +1,128 @@
----
+# This yaml describes our current checks.
 AllCops:
+  TargetRubyVersion: 2.6
+  SuggestExtensions: false
   NewCops: enable
+  Exclude:
+    - "bin/**/*"
+    - "vendor/**/*"
+
+Layout/CommentIndentation:
```

Comment:
My understanding is with this initialization Rubocop uses the idiomatic Ruby guide by default.  Therefore we only need to indicate the Enabled: false exclusions.  

How are we implementing plan to only check for complex methods?